src/datasources/tag/TagDataSource.ts

```diff
@@ -32,14 +32,20 @@ export class TagDataSource extends DataSourceApi<TagQuery> {
         refId: target.refId,
         fields: [
           { name: 'Time', values: [from, to], type: FieldType.time },
-          { name: 'Value', values: [target.constant, target.constant], type: FieldType.number },
+          { name: 'Value', values: [2.72, 3.14], type: FieldType.number },
```

Comment:
Can we pull these values out to constants?

Reply:
These aren't meaningful values - they'll be removed in the next change. (it's _e_ and π 😅)